pkg/k8sutil/deployments.go

```diff
@@ -384,3 +389,105 @@ func (k *K8sutil) CreateKibanaDeployment(baseImage, clusterName, namespace strin
 
 	return nil
 }
+
+// CreateCerebroDeployment creates a deployment of Cerebro
+func (k *K8sutil) CreateCerebroDeployment(baseImage, clusterName, namespace, cert string, imagePullSecrets []myspec.ImagePullSecrets) error {
+	replicaCount := int32(1)
+	component := fmt.Sprintf("elasticsearch-%s", clusterName)
+	deploymentName := fmt.Sprintf("%s-%s", cerebroDeploymentName, clusterName)
+
+	// Check if deployment exists
+	deployment, err := k.Kclient.ExtensionsV1beta1().Deployments(namespace).Get(deploymentName, metav1.GetOptions{})
+
+	if len(deployment.Name) == 0 {
+		logrus.Infof("Deployment %s not found, creating...", deploymentName)
+
+		deployment := &v1beta1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: deploymentName,
+				Labels: map[string]string{
+					"component": component,
+					"role":      "cerebro",
+					"name":      deploymentName,
+				},
+			},
+			Spec: v1beta1.DeploymentSpec{
+				Replicas: &replicaCount,
+				Template: v1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{
+							"component": component,
+							"role":      "cerebro",
+							"name":      deploymentName,
+						},
+					},
+					Spec: v1.PodSpec{
+						Containers: []v1.Container{
+							v1.Container{
+								Name:            deploymentName,
+								Image:           baseImage,
+								ImagePullPolicy: "Always",
+								Command: []string{
+									"bin/cerebro",
+									"Dconfig.file=/opt/cerebro/cfg/application.conf",
```

Comment:
This needs a `-` added. Based on the docker file I built, this is what I was testing with: 

```
"/usr/local/cerebro/bin/cerebro",
"-Dconfig.file=/opt/cerebro/cfg/application.conf",
```

Reply:
Done